src/fah/client/Unit.cpp

```diff
@@ -405,11 +405,11 @@ void Unit::setProgress(double complete, int total) {
   if (complete && total <= 0) return;
 
   double progress = complete ? (double)complete / total : 0;
```

Comment:
This code could be simplified to:

    double progress = (double)complete / total;